portfolio/src/main/webapp/fetcher.js

```diff
@@ -1,9 +1,10 @@
 //quick fetch function 
-async function strFetch(){
-    const gson = new Gson();
-    const servletText = (await fetch('/strservlet')).text(); //get the servlet text
+async function strFetch(){  
```

Comment:
Horizontal whitespace is needed "before any open curly brace (`{`)"

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

Reply:
Also, "On both sides of the double slash (`//`) that begins an end-of-line comment."